components/script/dom/performanceentry.rs

```diff
@@ -51,6 +51,14 @@ impl PerformanceEntry {
     pub fn name(&self) -> &DOMString {
         &self.name
     }
+
+    pub fn start_time(&self) -> f64 {
+        self.start_time
+    }
+
+    pub fn duration(&self) -> f64 {
```

Comment:
Not at the moment indeed. I thought it would be needed during implementation of the various timeline APIs.